packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1753,15 +2067,19 @@ export class MergeTree {
 		let fromSplit: IMergeBlock | undefined;
 		for (childIndex = 0; childIndex < block.childCount; childIndex++) {
 			child = children[childIndex];
-			const len = this.nodeLength(child, refSeq, clientId);
+			let len = this.nodeLength(child, refSeq, clientId);
 			if (len === undefined) {
-				// if the seg len in undefined, the segment
-				// will be removed, so should just be skipped for now
-				continue;
-			} else {
-				assert(len >= 0, 0x4bc /* Length should not be negative */);
+				if (this.breakTie(_pos, child, seq)) {
```

Comment:
this looks like a semantic change for what `breakTie` does. can you document the intent here? it seems like it's adding complexity that might be better represented in some other way.